src/net_processing.cpp

```diff
@@ -4604,15 +4637,15 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             std::sort(unique_parents.begin(), unique_parents.end());
             unique_parents.erase(std::unique(unique_parents.begin(), unique_parents.end()), unique_parents.end());
 
-            // Distinguish between parents in m_recent_rejects and m_recent_rejects_reconsiderable.
+            // Distinguish between parents in m_recent_rejects and RecentRejectsReconsiderableFilter().
```

Comment:
Shouldn't it be `RecentRejectsFilter()` or am I missing anything?

```suggestion
            // Distinguish between parents in RecentRejectsFilter() and RecentRejectsReconsiderableFilter().
```